src/gui/tray/Window.qml

```diff
@@ -139,6 +147,90 @@ ApplicationWindow {
         }
     }
 
+    Drawer {
+        id: fileDetailsDrawer
+        width: parent.width
+        height: parent.height
+        padding: 0
+        edge: Qt.RightEdge
+        modal: false
+        visible: false
+
+        background: Rectangle {
+            radius: Systray.useNormalWindow ? 0.0 : Style.trayWindowRadius
+            border.width: Style.trayWindowBorderWidth
+            border.color: Style.menuBorder
+            color: Style.backgroundColor
+        }
+
+        property var folderAccountState: ({})
+        property string fileLocalPath: ""
+        property int pageToShow: 0

```

Comment:
should you not be using `var` to preserve the `enum` type or I am just making this up ?
I fail to remember if it is possible to keep proper enum typing info into qml

Reply:
AFAIK `var` essentially makes use of QVariant under the hood, which should preserve enum types if declared to the metatype system

switched use to `var`